### PR TITLE
Monitor combat log for timeouts to make sure recording doesnt go nuts

### PR DESCRIPTION
**Pull request by prsauer**

### CODE REVIEW

File: packages/app/src/nativeBridge/modules/logsModule/index.ts [hunk 1]
```diff
@@ -225,4 +248,9 @@ export class LogsModule extends NativeBridgeModule {
   public handleParserError(_mainWindow: BrowserWindow, _error: Error) {
     return;
   }
+
+  @moduleEvent('on')
+  public handleLogReadingTimeout(_mainWindow: BrowserWindow, _wowVersion: WowVersion, _timeoutSeconds: number) {
```
Comment (atinylittleshell) [inline packages/app/src/nativeBridge/modules/logsModule/index.ts:253]: do we need to pass timeoutSeconds in if that's a constant? would it be more meaningful to pass in the actual delta time?

File: packages/app/src/nativeBridge/modules/logsModule/index.ts [hunk 2]
```diff
@@ -39,8 +39,31 @@ const bridgeState: {
   },
 };
 
+const READ_TIMEOUT_MS = 60000;
```
Comment (atinylittleshell) [inline packages/app/src/nativeBridge/modules/logsModule/index.ts:42]: is it more accurate to call it "timeout" or "idle"? it's not that something failed in IO, but simply no actual logs being written, right?

**Review by atinylittleshell — APPROVED** [review]

File: packages/app/src/nativeBridge/modules/logsModule/index.ts (hunk 2 above)
Comment (prsauer) [reply packages/app/src/nativeBridge/modules/logsModule/index.ts:42]: yeah it's a gray area -- from the perspective of the match we were expecting to find more logs but didn't (waiting for the activity to end timed out); from the perspective of reading the disk nothing is wrong we just don't have more data (the combat log writer is idle)

File: packages/app/src/nativeBridge/modules/logsModule/index.ts (hunk 1 above)
Comment (prsauer) [reply packages/app/src/nativeBridge/modules/logsModule/index.ts:253]: can swap to that, neither is strictly required for the callback to function correctly on the f/e side

**Pull request merged**